dll/win32/mstask/stubs.c

```diff
@@ -0,0 +1,133 @@
+
+#include "precomp.h"
+
+WINE_DEFAULT_DEBUG_CHANNEL(mstask);
+
+#define NET_API_STATUS DWORD
+typedef HANDLE ATSVC_HANDLE, SASEC_HANDLE;
+typedef struct _AT_INFO *LPAT_INFO;
+typedef struct _AT_ENUM_CONTAINER *LPAT_ENUM_CONTAINER;
+
+HRESULT
```

Comment:
The purpose of having C stubs is just to get us closer to the real implementation, or is it also related to the export situation somehow?

Reply:
I experimented with different attributes on the stubs in order to get the export rights, but it didn't work. But since I was already at it, I decded to just add them.